.github/workflows/release.yaml

```diff
@@ -0,0 +1,79 @@
+---
+name: Release
+
+run-name: release ${{ inputs.tag }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+permissions:
+  packages: write
+  id-token: write
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set env vars
+        run: |
+          echo PKG_VERSION="$(yarn -s run get-version)" >> $GITHUB_ENV
+          echo PKG_NAME="$(yarn -s run get-package-name)" >> $GITHUB_ENV
+          echo PKG_BASENAME="$(yarn -s run get-package-basename)" >> $GITHUB_ENV
+          echo VERSION="${{ inputs.tag }}" >> $GITHUB_ENV
```

Comment:
would be more readable to add an environment block on the job, to specify this variable (obviously can't be done for the others, which are computed...)

Reply:
done